service/src/test/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsServiceTest.java

```diff
@@ -919,6 +922,91 @@ void shouldAdjournHearing(HearingReListOption hearingReListOption, HearingStatus
             assertThat(caseData.getCancelledHearingDetails()).containsExactly(adjournedHearing);
             assertThat(adjournedHearingBundle.getValue().getHearingName()).isEqualTo(updatedDocumentBundleName);
         }
+
+        @Test
+        void shouldAdjournAndReListHearingAndUpdateDraftCaseManagementOrder() {
```

Comment:
Do you think we need an e2e for this functinality or is unit enough?

Reply:
I added integration tests and tested manage hearings scenarios